src/main.scss

```diff
@@ -20,43 +20,104 @@ $o-brand: 'internal';
 
 @import 'src/scss/colors';
 @import 'src/scss/component/main';
-@import 'src/scss/error';
-@import 'src/scss/layout/main';
+@import 'src/scss/layout';
 @import 'src/scss/overview/main';
 @import 'src/scss/status-labels';
 
-@include oFontsInclude(MetricWeb, regular);
-@include oFontsInclude(MetricWeb, semibold);
+// Layout.
+@include oLayout($opts: (
+	'layouts': ('query', 'docs'),
+	'features': ('sidebar-nav', 'linked-headings', 'typography')
+));
+
+// Tables.
 @include oTable($opts: ('responsive-overflow', 'lines'));
 
-html, body {
-	height: 100%;
-}
+// Header.
+@include oHeaderServices();
+
+// Footer.
+@include oFooterServicesWithLogo($logo: 'origami');
+
+// Forms.
+@include oFormsBaseFeatures();
+@include oFormsWideFeature();
+@include oFormsRadioCheckboxFeatures();
 
 .o-table-container,
 .o-table {
-    margin-bottom: oTypographySpacingSize(4);
+	margin-bottom: oTypographySpacingSize(4);
 }
 
-.registry {
-	@include oTypographySans(0);
+.registry-link--external {
+	@include oTypographyLinkExternal();
+}
 
-	.link {
-		@include oTypographyLink();
-	}
+.o-layout__aside-sidebar pre code {
+	white-space: normal;
+	word-break: break-all;
+}
+
+.registry-button {
+	width: 100%;
+}
+
+.registry-button--secondary {
+	@include oButtons(big, 'mono');
+	border-bottom: 1px solid oColorsGetPaletteColor('slate');
+}
+
+.o-forms {
+	// Padding overrides can be removed when the next major of o-forms is released.
+	padding-right: 0;
+	padding-left: 0;
+}
 
-	.link-external {
-		@include oTypographyLinkExternal();
+.registry-error {
+	position: relative;
+	margin: auto;
+	text-align: center;
+	max-width: oTypographyMaxLineWidth($scale: 1);
+}
+
+.registry-component-version-form {
+	margin-bottom: oTypographySpacingSize(2);
+	fieldset {
+		appearance: none;
+		border: 0;
+		padding: 0;
+		margin: 0;
+		min-width: 0;
 	}
 
-	blockquote {
-		@include oTypographyBlockquote();
+	legend {
+		@include oNormaliseVisuallyHidden();
 	}
+}
 
-	.secondary-button {
-		@include oButtons(big, mono);
-		width: 100%;
-		border-bottom: 1px solid oColorsGetPaletteColor('slate');
-		text-transform: uppercase;
+.registry-title span {
+	vertical-align: middle;
+}
+
+details {
+	margin-bottom: oTypographySpacingSize(4);
+	summary {
+		@include oTypographyItalic();
```

Comment:
I'm of the opinion that we shouldn't use italic, I also don't know why we are using italic.
If we want different styles from the paragraph inside the details element when it is open we can use `details[open] summary { font-weight: something}`. I tried using `text-decoration` but that seems to not work with summary elements inside details elements /shrug.

Reply:
I think I prefer italics, I think it works for secondary FAQ content, but don't feel too strongly. I've updated to the font weight on click approach...

Before:
![italics](https://user-images.githubusercontent.com/10405691/52639635-9afd2c00-2ecc-11e9-9d75-6b8acc1d796c.gif)

After:
![bold](https://user-images.githubusercontent.com/10405691/52639627-96387800-2ecc-11e9-99d8-ba7f31d65082.gif)